src/pages/Asset/index.tsx

```diff
@@ -407,18 +369,14 @@ export type AssetProps = RouteComponentProps<{ vTokenId: VTokenId; marketId: str
 
 const Asset: React.FC<AssetProps> = ({
   match: {
-    params: { vTokenId, marketId },
+    params: { vTokenId },
   },
 }) => {
-  const { account } = useContext(AuthContext);
   const vToken = getVBepToken(vTokenId);
 
-  // TODO: fetch actual values (see VEN-546)
```

Comment:
is that TODO not necessary anymore?

Reply:
It isn't. Check the line after this comment, you'll see a variable set with a fixed value which has now ben removed and that was the one that needed to be fetched originally.